ArduCopter/RC_Channel.cpp

```diff
@@ -561,6 +562,21 @@ void RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const aux_sw
             }
             break;
             
+        case AUX_FUNC::DISARM:
+            switch (ch_flag) {
+            case HIGH:
+                AP::arming().disarm(AP_Arming::Method::AUXSWITCH);
```

Comment:
We could consider adding another `Method` to disambiguate, but I don't think it's required.